src/types/markers.js

```diff
@@ -337,6 +337,7 @@ export type NetworkPayload = {|
   count?: number, // Total size of transfer, if any
   status: string,
   cache?: string,
+  contentType?: string,
```

Comment:
I believe the type is incorrect, because this can be null.
Also it would be nice to have a comment explaining the various possible cases (eg: missing in profiles from old Firefox, etc).

Reply:
Yes you are right, thanks!